workflows/prognostic_c48_run/tests/machine_learning_mocks.py

```diff
@@ -36,16 +35,20 @@ def get_mock_predictor(model_predictands: str = "tendencies") -> fv3fit.Predicto
     if model_predictands == "tendencies":
         output_variables = ["dQ1", "dQ2", "dQu", "dQv"]
     elif model_predictands == "rad_fluxes":
-        output_variables = ["downward_shortwave", "net_shortwave", "downward_longwave"]
+        output_variables = [
+            "total_sky_downward_shortwave_flux_at_surface",
```

Comment:
I changed the names of the radiative flux variables in the test to match those in the state, since the MLStepper now uses the state to check which predictions are state updates.